crypto/CMakeLists.txt

```diff
@@ -1859,6 +1859,7 @@ add_library(${lib_name} STATIC ${${lib_name}_libsrcs})
 target_include_directories(${lib_name} PRIVATE
   ${CMAKE_CURRENT_SOURCE_DIR}/include
   ${CMAKE_CURRENT_BINARY_DIR}/include # buildinf.h, internal/[bn|dso]_conf.h
+  ${CMAKE_CURRENT_SOURCE_DIR} # arm_arch.h
```

Comment:
On arm, it needs to be able to include arm_arch.h, which is in the crypto folder.